client/src/containers/FormBuilderLoader/createIdentifier.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Creates a dot-separated identifier for forms generated
+ * with schemas (e.g. FormBuilderLoader)
+ *
+ * @param props
+ * @returns {string}
+ */
+const createIdentifier = (props) => {
+  const { identifier, schema } = props;
```

Comment:
maybe deconstruct in the params? could do `{ identifier, schema = {} }` so can remove at least one exists check below